libloki/test/node/service_nodes_test.js

```diff
@@ -0,0 +1,58 @@
+const ServiceNode = require('../../service_nodes');
+const { expect } = require('chai');
+
+describe('ServiceNodes', () => {
+  describe('#consolidateLists', () => {
+    it('should throw when provided a non-iterable list', () => {
+      expect(() => ServiceNode.consolidateLists(null, 1)).to.throw();
```

Comment:
Are we using `expect` or `assert` as a general test pattern? i remember having a discussion saying we stick with one or the other but i can't remember if we decided to use `assert` pattern or the `expect` pattern. @BeaudanBrown thoughts?

Reply:
I have been using assert over expect because expect was only used in a couple files, don't think there are any real differences between them? So maybe lock in one for style?

Reply:
I just copied the first node test I could find under `test/app` and it was using expect :)
(My c++ tinted 2 cents, `expect` sounds more unit-testy than `assert`)